node/resolvers/benefits/index.ts

```
@@ -66,6 +66,10 @@ export const getBenefits = async (
       },
     ],
   }
-  const benefitsData = await checkout.simulation(requestBody)
-  return path(['ratesAndBenefitsData', 'teaser'], benefitsData) || []
+  try {
+    const benefitsData = await checkout.simulation(requestBody)
+    return path(['ratesAndBenefitsData', 'teaser'], benefitsData) || []
+  } catch (e) {
+    return 'error'
+  }
```

In case you don't want to return a string `'error'`, you have this two options

You can remove the try/catch and Instead of
```javascript
let result = flatten(await Promise.all(items?.map(item => getBenefits(item.itemId, ctx))))
let benefitsWithoutError = result.filter(item => item !== 'error')
```

you can do something like this
```javascript
const promises = items?.map((item) => getBenefits(item.itemId, ctx));
let result = flatten(await Promise.all(promises.map((p) => p.catch((e) => e))));
let benefitsWithoutError = result.filter((result) => !(result instanceof Error));
```

If you think this method is too verbose you can simply return the error itself

```javascript
} catch (e) {
    return e
 }
```

and then filter by type

```javascript
let benefitsWithoutError = result.filter((result) => !(result instanceof Error));
```

I prefer the verbose method because if someone is going to call this function they should assume that it will throw an error when an error happens